src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/InputEmptyDefault.java

```diff
@@ -0,0 +1,89 @@
+public class InputEmptyDefault {
+    void method1(int a) {
+        switch (a) {}
+        switch (a) {default: ; } // no violation
+        switch (a) {default: {}} // violation
+        switch (a) {
+            default:
+        }
+        switch (a) {
+            default:             // violation
+            {}
+        }
+        switch (a) {
+            default:             // violation if checking statements
+            { // text
+            }
+        }
+    }
+
+    void method2(int a) {
+        switch (a) {
+            case 1:a++;
+            case 2:a++;
+            default:             // no violation
+                switch (a) {
+                    default: {   // violation if checking for statements
+
+                    }
+                }
+        }
+    }
+
+    void method3(int a, int b) {
+        switch (a) {
+            case 1: break;
+            default: {} method2(a);  // violation
+        }
+
+        switch (b) {
+            case 2: break;
+            default: method2(b); {}  // no violation
+        }
+
+        switch (a+b) {case 1: break; default: {} ; } // violation
+    }
+
+    void method4(int a, int b) {
+        switch (a) {
+            case 1:
+            default: {}    // violation
+        }
+
+        switch (b) {
+            case 1:
+            default:       // no violation
+        }
+
+        switch (a+b) {
+            default:       // no violation
+            case 1: { }
```

Comment:
@romani Please confirm you are ok with this having no violation, just because `default` is before `case` and only `default` is set for validation.
This type of scenario isn't that apparent to me that it will happen.
We may need some clarification in xdoc for this.